kvdb/sqlbase/sqlerrors_no_postgres.go

```diff
@@ -0,0 +1,10 @@
+//go:build !kvdb_postgres
+// +build !kvdb_postgres
```

Comment:
I think we should start to remove the old style build tags everywhere, or at least not use them in new code anymore.